loafwallet/src/Strings/tr.lproj/Localizable.strings

```diff
@@ -232,1169 +232,1169 @@
 /* Regular fee */
 "FeeSelector.regular" = "Düzenli";
 
-/* Fee Selector regular fee description */
-"FeeSelector.regularLabel" = "Estimated Delivery: 2.5 - 5+ minutes";
+/* Ücret Seçici normal ücret açıklaması */
```

Comment:
@JasperHeist  Oh I see its confusing but the process is to leave the Comments in english (revert line 235) for example